docs/ONNX_Runtime_Perf_Tuning.md

```diff
@@ -67,12 +67,12 @@ import onnxruntime as rt
 sess_options = rt.SessionOptions()
 
 sess_options.intra_op_num_threads=2
-sess_options.enable_sequential_execution=True
+sess_options.execution_mode=rt.ExecutionMode.SEQUENTIAL
 sess_options.set_graph_optimization_level(2)
 ```
 * sess_options.intra_op_num_threads=2 controls how many thread do you want to use to run your model
-* sess_options.enable_sequential_execution=True controls whether you want to run operators in your graph sequentially or in parallel. Usually when your model has many branches, set this option to false will give you better performance.
-* When sess_options.enable_sequential_execution=False, you can set sess_options.inter_op_num_threads to control the
+* sess_options.execution_mode=rt.ExecutionMode.SEQUENTIAL controls whether you want to run operators in your graph sequentially or in parallel. Usually when your model has many branches, set this option to false will give you better performance.
+* When sess_options.execution_mode=rt.ExecutionMode.PARALLEL, you can set sess_options.inter_op_num_threads to control the
```

Comment:
set -> setting.
Should it say to set it to PARALLEL instead of 'false'?

Reply:
fixed